.pre-commit-config.yaml

```diff
@@ -1,30 +1,13 @@
 repos:
-  - repo: https://github.com/ambv/black
-    # black >= 24 is incompatible with reorder-python-imports
-    # See https://github.com/psf/black/issues/4175
-    rev: "23.12.1"
+  - repo: https://github.com/astral-sh/ruff-pre-commit
+    # Ruff version.
+    rev: v0.7.4
     hooks:
-      - id: black
-  - repo: https://github.com/asottile/reorder_python_imports
-    rev: "v3.14.0"
-    hooks:
-      - id: reorder-python-imports
-        args: ["--py39-plus"]
-  - repo: https://github.com/asottile/pyupgrade
-    rev: "v3.19.0"
-    hooks:
-      - id: pyupgrade
-        args: ["--py39-plus"]
-  - repo: https://github.com/pycqa/flake8
-    rev: "7.1.1"
-    hooks:
-      - id: flake8
-        language_version: python3
-        additional_dependencies:
-          # NB: autoupdate does not pick up flake8-bugbear since it is a
-          # transitive dependency. Make sure to update flake8-bugbear
-          # manually on a regular basis.
-          - flake8-bugbear==24.10.31
```

Comment:
Here's the list of rules enabled by default: https://docs.astral.sh/ruff/configuration/#configuring-ruff.
Follow https://docs.astral.sh/ruff/linter/#rule-selection to re-enable bugbear.

Reply:
Ah.. nevermind. I didn't see the config at the very bottom of the PR.